openmdao/core/group.py

```diff
@@ -108,6 +108,24 @@ def _subsystem(self, name):
             s = s._subsystems[part]
         return s
 
+    def find_subsystem(self, name):
+        """
+        Returns a reference to a named subsystem that is a direct or an indirect
+        subsystem of the this system.  Raises an exception if the given name
+        doesn't reference a subsystem.
+
+        Args
+        ----
+        name : str
+            Name of the subsystem to retrieve.
+
+        Returns
+        -------
+        `System`
+            A reference to the named subsystem.
+        """
+        return self._subsystem(name)
```

Comment:
can you replace _subsystem() everywhere with find_subsystem() instead of having find_subsystem() just forward to _subsystem()?